packages/env-tests/src/monorepoRun.ts

```diff
@@ -21,10 +20,32 @@ function runTests() {
   const mnemonic = process.env.MNEMONIC!
   const reserveSpenderMultiSigAddress = process.env.RESERVE_SPENDER_MULTISIG_ADDRESS
 
+  const defaultTokensToTest = ['cUSD']
+  let stableTokensToTest: string[]
+  if (!process.env.STABLETOKENS) {
+    stableTokensToTest = defaultTokensToTest
+  } else {
+    const tokens = process.env.STABLETOKENS.split(',')
+
+    for (let token of tokens) {
+      if (!StableTokenToRegistryName[token]) {
+        throw new Error(`Invalid token: ${token}`)
+      }
+    }
+    stableTokensToTest = tokens
```

Comment:
It might be more readable to just have a consistent pipeline, something like:

```typescript
const tokens  = process.env.STABLETOKENS ? process.env.STABLETOKENS.split(',') : defaultTokensToTest
if (tokens.find(token => !StableTokenToRegistryName[token])) throw new Error()

This avoids mutable `let`
```

Reply:
I have changed it accordingly